File: help-car/model/Prestador.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace help_car.model
{
    [Table("tbprestador")]
    public class Prestador
    {

        [Column("id")]
        public int? id { get; set; }
        public string? nome { get; set; }
        public string? cnpj { get; set; }
        public DateTime? data_nascimento { get; set; }
        public string? email { get; set; }
        public string? senha { get; set; }
        public string? nome_do_comercio { get; set; }
        public string? servico_do_comercio { get; set; }
     
        public double? valor_servico { get; set; }


}
}

File: front-end/src/app/models/prestador.ts
export class Prestador {
  id: number ;
  nome: string ;
  cnpj: string ;
  email: string ;
  senha: string ;
  nome_do_comercio: string ;
  servico_do_comercio: string ;
  valor_servico: number ;



  public constructor(data:any) {
    if (data) {
      this.id = Number(data.id);
      this.nome = String(data.nome);

    }else{
      this.id = 0;
      this.nome =  '';

    }
  }
}
